packages/lib/src/nav-tabs/Tab.tsx

```diff
@@ -5,57 +5,55 @@ import DxcFlex from "../flex/Flex";
 import NavTabsPropsType, { TabProps } from "./types";
 import NavTabsContext from "./NavTabsContext";
 import DxcIcon from "../icon/Icon";
+import DxcInset from "../inset/Inset";
 
-const TabContainer = styled.div<{ active: TabProps["active"] }>`
-  align-items: stretch;
-  border-bottom: var(--border-width-s) var(--border-style-default)
-    ${(props) => (props.active ? "var(--border-color-primary-stronger)" : "transparent")};
-  padding: var(--spacing-padding-xs);
+const TabContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
 `;
 
-const Tab = styled.a<{
+const TabLink = styled.a<{
   disabled: TabProps["disabled"];
-  hasIcon: boolean;
   iconPosition: NavTabsPropsType["iconPosition"];
 }>`
   box-sizing: border-box;
   display: flex;
-  flex-direction: ${(props) => (props.hasIcon && props.iconPosition === "top" ? "column" : "row")};
+  flex-direction: ${({ iconPosition }) => (iconPosition === "top" ? "column" : "row")};
   justify-content: center;
   align-items: center;
   gap: var(--spacing-gap-xs);
-  height: ${(props) => (props.hasIcon && props.iconPosition === "top" ? "78px" : "100%")};
+  height: ${({ iconPosition }) => (iconPosition === "top" ? "78px" : "100%")};
```

Comment:
This has the unintended effect that the height can change even if none of the tabs has an icon.

Reply:
I know, but it makes zero sense if you have a tab without an icon and `iconPosition` set on purpose to `"top"` right?

Reply:
Also, I understand that if you set `iconPosition` to top for the whole component is that you anticipate that it can go with an icon, so if the height it's already set it won't do a size "jump".